### PR TITLE
JanDeDobbeleer.OhMyPosh version 3.175.1

### PR DESCRIPTION
**Pull request by JanDeDobbeleer**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26578)

### CODE REVIEW

File: manifests/j/JanDeDobbeleer/OhMyPosh/3.175.1/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ShortDescription: Prompt theme engine for any shell
+Tags:
+- Console
+- Command-Line
+- Shell
+- Command-Prompt
+- PowerShell
+- WSL
+- Developer-Tools
+- Utilities
```
Comment (vedantmgoyal2009) [inline manifests/j/JanDeDobbeleer/OhMyPosh/3.175.1/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml:19]:
```suggestion
Tags:
- console
- command-line
- shell
- command-prompt
- powershell
- wsl
- developer-tools
- utilities
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags

**Comment by JanDeDobbeleer** [comment]
I'll adjust this in the next version as the next was already approved.

File: manifests/j/JanDeDobbeleer/OhMyPosh/3.175.1/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml (hunk 1 above)
Comment (JanDeDobbeleer) [reply manifests/j/JanDeDobbeleer/OhMyPosh/3.175.1/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml:19]: If this is a convention it should fail automatic validation. If it's a suggestion it should not block a merge.

Comment (vedantmgoyal2009) [reply manifests/j/JanDeDobbeleer/OhMyPosh/3.175.1/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml:19]: It is neither of them. If you have read my comment, it's just a best practice.

**Pull request closed without merging**